sdk/keyvault/azure-security-keyvault-certificates/CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Release History
 
-## 4.1.0-beta.1 (Unreleased)
+## 4.0.2 (2020-03-11)
```

Comment:
Shouldn't this be 4.0.1 to match the version?

Reply:
yeah, that's a good catch.
But not releasing the patch anymore. So, closing this PR.